pages/common/column.md

```diff
@@ -0,0 +1,19 @@
+# column
+
+> Format standard or file input into multiple columns, rows are filled before columns.
+
+- Output is formatted for a 10 columns wide display:
```

Comment:
addressed in https://github.com/kidpixo/tldr/commit/207d2d38d7a8ccb55764918d5df4dd3f3eaac114